libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/InstructionView.java

```diff
@@ -222,13 +221,12 @@ public void onChanged(@Nullable Boolean isOffRoute) {
   public void update(RouteProgress routeProgress) {
     if (routeProgress != null && !isRerouting) {
       InstructionModel model = new InstructionModel(getContext(), routeProgress, language, unitType);
-      updateViews(model);
-      updateTextInstruction(model);
+      updateDataFromInstructionModel(model);
```

Comment:
Minor comment - what about `updateDataFromInstruction(model)` and `updateDataFromBannerInstruction(model)`? I think it'd be better for consistency and it reads well. What do you think?

Reply:
@Guardiola31337 yeah I like that naming a lot better, thanks 💯